src/components/SVGViewer/__tests__/SVGViewer.test.tsx

```diff
@@ -35,10 +37,20 @@ describe('SVGViewer', () => {
         <SVGViewer documentId={0} handleCancel={handleCancel} />
       </ClientSDKProvider>
     );
-    const closeButton = wrapper.find('[data-test-id="close-svgviewer-btn"]');
+    const closeButton = wrapper.find(CANCEL_BUTTON_ATTR);
     expect(closeButton).toBeTruthy();
     closeButton.first().simulate('click');
 
     expect(handleCancel.called).toBeTruthy();
   });
+
+  it('Renders without exploding with a file', () => {
+    const wrapper = mount(
+      <ClientSDKProvider client={sdk}>
+        <SVGViewer file="" />
```

Comment:
Does it behavior that we really expect? We need to provide non-empty string here, don't we?

Reply:
Strictly speaking `string` typescript type contains empty strings

Reply:
I know but should prop type here be more accurate, cause there is no sense to put empty string here

Reply:
doesn't seem it's that easy to type it
https://stackoverflow.com/questions/46253340/how-to-write-a-string-type-that-does-not-contain-an-empty-string-in-typescript

or you mean just pass non-empty string in a test?

Reply:
No, I meant that empty string will break `componentDidUpdate` lifecycle method if we provide `string` as a type for `file` prop. So if we can't define more specific type we should redo `componentDidUpdate` section (check if `file` prop is not `undefined`)

Reply:
wait isnt it checking if it is undefined right now? do u mean we should check if its non-empty (contains `svg` or someting)

Reply:
```
(curDocId && curDocId !== prevDocId) ||
(currFile && currFile !== prevFile)
```
it checks for an empty string, no?

Reply:
yes, but in case of changing `documentId` to `file` prop with an empty string component will still render svg related to `documentId`, which is not what we're expecting. We need to check `curDocId` and `currFile` for `undefined` value